tests/Unit/API/REST/V3/Routes/Donors/GetDonorRouteTest.php

```diff
@@ -314,18 +314,17 @@ public function testGetDonorShouldRedactAnonymousDonor()
         $data = $response->get_data();
 
         $this->assertEquals(200, $status);
-        $this->assertEquals(0, $data['id']);
 
         $anonymousDataRedacted = [
-            //'id', // This property is Checked above...
+            'id',
             'name',
             'firstName',
             'lastName',
             'prefix',
         ];
 
         foreach ($anonymousDataRedacted as $property) {
-            $this->assertEquals(__('anonymous', 'give'), $data[$property]);
+            $this->assertFalse(isset($data[$property]));
```

Comment:
We also need to rollback this test to keep the consistency with this: https://github.com/impress-org/givewp/pull/7939#discussion_r2114504754

Reply:
Resolved https://github.com/impress-org/givewp/pull/7939/commits/5c8431ee5647020e8835bf7e55ea35cb691ef04a